package.json

```diff
@@ -18,7 +18,7 @@
     "start": "nest start",
     "__predev": "npm run generate-database-types",
     "prestart:debug": "npm run generate-database-types",
-    "dev": "dotenv -e .env/.env.local nest start --watch",
+    "dev": "dotenv -e .env/.env.local nest start -- --watch",
```

Comment:
Do we need the extra `--` ?

Reply:
yes, i think npm doesn't otherwise correctly pass the --watch all the way to the next start commend.
Without this fix the server doesn't auto restart
with this fix the server does correctly restart